versions.tf

```diff
@@ -1,17 +1,9 @@
 terraform {
   required_version = ">= 0.12.10"

```

Comment:
@BostjanBozic thanks for submitting this pull request! I believe we need to set this to `">= 0.13.0"` as the new provider syntax (with the source + version info) was introduced in Terraform 0.13.

Reply:
@fabiendelpierre thank you for fast review. It definitely makes sense, I have pushed the change.